tests/actions/test.subs-actions.js

```diff
@@ -107,6 +114,14 @@ describe('subscription actions', function() {
       });
     });
 
+    it('should dispatch an error on tokenization failure', function() {
+      var jquery = helpers.fakeJquery();
+      createSubscription(jquery.stub, FakeBraintreeClientError);
+      var action = dispatchSpy.firstCall.args[0];
+      assert.deepEqual(action,
+                       appActions.error('Braintree tokenization error'));
+    });
```

Comment:
The fact that there is a duplicate test for this makes me think we need a separate helper to wrap up the tokenization and common failure dispatch. It could be used maybe like this as a promise:
````
addSubscription(...) {
  return dispatch => {
    tokenizeCard(dispatch, braintreeToken, creditCard).then(nonce => {
      jquery.ajax(...)  // add the subscription
    });
  }
}
````

this way, we can make sure the helper always dispatches the tokenization failure.

If not in this patch, maybe file an issue for it?